druid/src/widget/align.rs

```diff
@@ -121,7 +121,8 @@ impl<T: Data> Widget<T> for Align<T> {
         let extra_height = (my_size.height - size.height).max(0.);
         let origin = self
             .align
-            .resolve(Rect::new(0., 0., extra_width, extra_height));
+            .resolve(Rect::new(0., 0., extra_width, extra_height))
+            .floor();
```

Comment:
```suggestion
            .expand();
```
This better matches the behaviour in `BoxConstraint`.